modules/tensor_mechanics/include/materials/StressUpdateBase.h

```diff
@@ -19,6 +19,22 @@
 // Forward declaration
 class StressUpdateBase;
 
+/**
+ * TangentCalculationMethod is an enum that determines the calculation method for the tangent
+ * operator. ELASTIC uses the elasticity tensor as the tangent operator: J = C. The elasticity
+ * tensor does not need to be provided by the StressUpdateBase models in this case. FULL calculates
+ * the full tangent operator tensor in each inherited class. The full tangent operator is then
```

Comment:
I just noticed that this comment is wrong too.  Make it like the other one you fixed up.

Reply:
Yeah, something got screwy with the commit...